src/main/webapp/script.js

```diff
@@ -1,3 +1,79 @@
+/*
+    These functions are strictly being used for the user's profile
+    to ensure the associated store is being displayed rather than the user's
+    email next to each review
+*/
+function determineLoginUser() {
+  fetch('/get-login-info').then(response => response.json()).then((isLoggedIn) => {
+    const element = document.getElementById('form');
+    const textElement = document.createElement('span');
+    const form = document.createElement('form');
+    form.method='GET';
+    form.action='/login';
+    const button = document.createElement('button');
+    if (!isLoggedIn) {
+      button.innerHTML = "Login";
+    } else {
+      displayEmailUser();
+      button.innerHTML = "Logout";
+    }
+    form.appendChild(button);
+    element.appendChild(textElement);
+    element.appendChild(form);
+  }).catch(error => {
+    console.error('There has been a problem with your operation:', error);
+  });
+}
+
+function displayEmailUser() {
+  fetch('/get-email').then(response => response.json()).then((email) => {
+    document.getElementById('email').innerText = email;
+   loadCommentsUser(email);
+  }).catch(error => {
+    console.error('There has been a problem with your operation:', error);
+  });
+}
+
+function loadCommentsUser(email) {
+  var url = new URL('/comment', "https://" + window.location.hostname);
+  var params = {email: email};
+  url.search = new URLSearchParams(params).toString();
+  fetch(url).then(response => response.json()).then((drinks) => {
+    const ratingListElement = document.getElementById('list');
+
+    drinks.forEach((drink) => 
+    {
+      ratingListElement.appendChild(createListElementUser(drink));
+    })
+
+  });
+}
+
+/** Creates an <li> element containing text as "store, drink, rating, and content". */
+function createListElementUser(drink) {
+  const ratingElement = document.createElement('li');
+  ratingElement.className = 'drink';
+
+  const drinkElement = document.createElement('span');
+  var store = idToStore(drink.store)
+  drinkElement.innerText = store +", "+drink.drink + ", " + drink.rating + "/5, " + drink.content;
+
+  ratingElement.appendChild(drinkElement);
+  return ratingElement;
+}
+/**This function turns the store id into a readable store */
```

Comment:
nit: for consistency with the rest of the code, leave a space between consecutive functions.